File: ServerWApp/Models/Kitchen.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ServerWApp.Models
{
    public class Kitchen
    {
        public int KitchenId {get; set;}
        public string? KitchenAd {get;set;}
        public string? Images {get;set;}
        public List<Restorant>? Restorants {get;set;}
    }
}

File: ng-app/src/app/models/kitchen.ts
import { Restorant } from "./restorant";

export class Model{
  // kitchens : Array<Kitchen>;
  // KitchenAd:string;
  kitchens: Array<Kitchen>;
  // constructor(){
  //   this.kitchens= [

  //     new Kitchen(1, "Pide"),
  //     new Kitchen(2, "Pizza"),
  //     new Kitchen(3, "Lahmacun"),
  //     new Kitchen(4, "Kebap")

  // ]
  // }
}

export class Kitchen{
  kitchenId: number;
  kitchenAd: string;
  images: string;
  restorants: Restorant[];


  constructor (KitchenId:number, KitchenAd: string, İmages: string){

    this.kitchenId= KitchenId;
    this.kitchenAd = KitchenAd;
    this.images=İmages;
    }
    // this.isActive= isActive;



}
